modules/crowdfundings/lib/updateUserOnMailchimp.js

```diff
@@ -32,6 +35,26 @@ module.exports = async ({userId, pgdb}) => {
       userId: userId,
       membershipTypeId: membershipTypeBenefactor.id
     }) : false
+
+    const enforcedNewsletterSubscriptions = isNew
+      ? {
+        // Autosubscribe free newsletters when user is new.
+        [MAILCHIMP_INTEREST_NEWSLETTER_PROJECTR]: true
+      }
+      : hasJustPaid
```

Comment:
what if it's not the users first payment and she already unsubscribed from the list? you should check how many pledges the users has and use something like `firstPledge`

Reply:
Good catch. I'm now checking against `!!hasJustPaid && hasPledge && pledges.length === 1` to make sure we only autosubscribe when the first pledge has just been paid.